api/v1alpha1/contour_types.go

```diff
@@ -266,6 +266,10 @@ type ProviderLoadBalancerParameters struct {
 	// +unionDiscriminator
 	// +kubebuilder:default=AWS
 	Type LoadBalancerProviderType `json:"type,omitempty"`
+
+	// loadBalancerIP contains IP for LoadBalancer service type, optional
+	// +optional
+	LoadBalancerIP string `json:"loadBalancerIP,omitempty"`
```

Comment:
would be good to add some validation that this string is in fact an ip address (here or in the validation package) so we can give feedback to users early if they have something wrong, and some min/max length validation on this string

Reply:
Added `// +kubebuilder:validation:Format=ipv4` to have strict formating